src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User alice = new User();
+    private User bob = new User();
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        alice.setLogin("Alice");
+        alice.setPassword("123Alice");
+        alice.setAge(18);
+
+        bob.setLogin("Bob");
+        bob.setPassword("123Bob");
+        bob.setAge(22);
+    }
+
+    @Test
+    void register_age_less_than_18_notOk() {
```

Comment:
```suggestion
    void register_ageLessThan18_notOk() {
```
same logic for other methods names